eZ/Publish/Core/FieldType/Image/ImageStorage/Gateway.php

```diff
@@ -59,6 +59,14 @@ abstract public function removeImageReferences($uri, $versionNo, $fieldId);
      */
     abstract public function countImageReferences($uri);
 
+    /**
+     * Returns if there is reference to the given $uri.
```

Comment:
I think this should say:
```suggestion
     * Returns true if there is reference to the given $uri.
```
right?